GameDesign/Assets/Scripts/MonoBehaviours/LevelMonobehaviour.cs

```diff
@@ -54,12 +57,20 @@ void Start()
         }
         manager.SetTotalSpawns(maskedCustomers, maskedWeightsDictionary, unmaskedDictionary);
         manager.SetPrefabs(unmaskedPrefabsDictionary, maskedPrefabsDictionary);
-        timerManager.StartTimer();
+        manager.SetWaves(wavePercentages);
+
+        if (manager.SanityCheck())
+        {
+            timerManager.StartTimer();
```

Comment:
if we want to start the level afterwards, like after a loading phase. we will have to move this. probably let it be called externally to signal the start of the level. maybe  if we think synchronizing loading is necessary we could have a manager just to check the signals of all what we deem necessary attributes